vcbuild.bat

```diff
@@ -14,7 +14,7 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=Release
 set target=Build
-set target_arch=x86
+set target_arch=x64
```

Comment:
At this point I'm not sure it's necessary to detect whether the OS is 64-bit, defaulting to 64-bit seems reasonable.